CHANGES/8010.doc

```diff
@@ -1,2 +0,0 @@
-On the `CHANGES/README.rst <https://github.com/aio-libs/aiohttp/tree/master/CHANGES/README.rst>`_ page,
```

Comment:
@Dreamsorcerer did you decide not to include this fragment into the resulting changelog? Why? Now, this change note only exists on `master` and shows up in the changelog draft for 4.0: https://docs.aiohttp.org/en/latest/changes.html.

Reply:
Also, removal of this fragment didn't happen in `3.10` either: https://github.com/aio-libs/aiohttp/commit/0124dc58f4a06a7316925e4c10e00033cabe1d2d.

Reply:
@Dreamsorcerer I checked the fragment list and this seems to be the only one that disappeared mysteriously.

Reply:
Ah, not sure. Must have done something weird.